client/my-sites/checkout/src/components/test/secondary-cart-promotions.tsx

```diff
@@ -41,6 +41,7 @@ describe( 'SecondaryCartPromotions', () => {
 							<SecondaryCartPromotions
 								responseCart={ responseCartWithRenewal }
 								addItemToCart={ jest.fn() }
+								isPurchaseRenewal={ true }
```

Comment:
I believe this is correct, since I moved isPurchaseRenewal outside of the component which caused the test to be lacking this data (since I never added the new prop to the test)